MAINTENENCE.md

```diff
@@ -0,0 +1,31 @@
+
+
+The Alpine image contains the SHA256 checksum for that version as listed on the SDK [documentation page](https://cloud.google.com/sdk/downloads#versioned).
+
+```bash
+export TAG_VER=153.0.0
+git add -A
+git commit -m "Updating version $TAG_VER"
+git push
+git tag $TAG_VER
+git push --tags
+```
+
+A github [pre-commit](hooks/pre-commit) hook is also provided which will build the alpine image with the SDK provided. 
```

Comment:
interesting idea but I have doubts if pre-commit hooks are a great idea. I left a comment about it below about how people have been doing it in open source.

Ideally this document should be just 3-4 lines.